docs/overview.rst

```diff
@@ -194,12 +183,12 @@ Here are some good ones:
 
 * ``breakpad_resource.crash_save.time``
 
-  Timing. This is the time it took to save the crash to S3.
+  Timing. This is the time it took to save the crash to Google Cloud Storage.
```

Comment:
This section is bunk. I want to update Antenna to Markus 5.0.0 which includes a filter that allows us to specify all the metrics in a file, enforce that only those are emitted, and then auto-document them. I'll get to that eventually. Here's the equivalent in Eliot:

https://mozilla-eliot.readthedocs.io/en/latest/metrics.html

https://github.com/mozilla-services/eliot/blob/main/eliot/eliot_metrics.yaml